From 9cdb933367b2cb266ffef1b98e81775780ddff83 Mon Sep 17 00:00:00 2001
From: Cesar DE OLIVEIRA <cesarx.de.oliveira@intel.com>
Date: Mon, 12 Nov 2012 17:27:25 +0100
Subject: [PORT FROM MAIN] Modem control driver: Modifying right access on device

BZ: 40361

As there is a new device used for modem control, right must be modified
in order to authorize the access to MMgr.

Change-Id: Ic3dd63f3fce8abaeff23a5874f0c1100be190071
Orig-Change-Id: I825192a247f0ca78425b5b9aa43a41977129aadc
Signed-off-by: Edward Marmounier <edward.marmounier@intel.com>
Reviewed-on: http://android.intel.com:8080/88978
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Robert, Denis <denis.robert@intel.com>
Tested-by: Robert, Denis <denis.robert@intel.com>
Reviewed-by: Berat, FredericX <fredericx.berat@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Ravindran, Arun <arun.ravindran@intel.com>
Reviewed-by: Shen, JianchunX <jianchunx.shen@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 rootdir/ueventd.rc |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index d1f58b1..ef4da15 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -76,12 +76,13 @@
 /dev/mtp_usb              0660   root       mtp
 /dev/usb_accessory        0660   root       usb
 /dev/tun                  0660   system     vpn
-/dev/mdmTrace             0660   root       radio
 /dev/ttyACM*              0660   root       radio
 /dev/ttyPTI1              0660   root       radio
 /dev/gsmtty*              0660   root       radio
 /dev/ttyMFD*              0660   root       radio
 /dev/ttyGS0               0660   root       radio
+/dev/mdmTrace             0660   root       radio
+/dev/mdm_ctrl             0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

